<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_1_9_insemenation</class>
 <widget class="QWidget" name="Form_1_9_insemenation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Общие сведения</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Номер животного:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Текущая лактация:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Дни лактации:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_number">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_current_lactation">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_days_lactation_2">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Предыдущая жара:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Предыдущее осеменение:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Статус репродуктивности:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_previous_heat">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_previous_insemenation">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_status_reproduction">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Ожидаемый отел:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_wait_otel">
               <property name="text">
                <string>---</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Животное 1 из 1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>История</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Осеменение</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Тип осеменения:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QRadioButton" name="radioButton_AI">
                    <property name="text">
                     <string>Искуственное осеменение</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_diy_ai">
                    <property name="text">
                     <string>Частное искуственное осеменение</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_sire">
                    <property name="text">
                     <string>Самец</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_period">
                    <property name="text">
                     <string>Периодическое</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioButton_implant_embrion">
                    <property name="text">
                     <string>Имплантация эмбриона</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Дата осеменения:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEdit_datr_insemenation"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Самец:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_sires">
                  <item>
                   <property name="text">
                    <string>sire3274</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_sire"/>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_sire_select">
                  <property name="text">
                   <string>Выбрать</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Персона: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_persons">
                  <item>
                   <property name="text">
                    <string>unknown</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Группа:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_group">
                  <item>
                   <property name="text">
                    <string>Automatic</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Комментарий:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEdit_remarks"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="pushButton_previous">
          <property name="text">
           <string>Предыдущий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_next">
          <property name="text">
           <string>Следующий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save_all">
          <property name="text">
           <string>Сохранить все</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="text">
           <string>Закрыть</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
